### PR TITLE
Implements example to use for write-once-run-everywhere blog post

### PR DESCRIPTION
**Pull request by elijahbenizzy**
Note that this is similar to the more complex feature engineering post, just meant to be bite-sized and demonstrate a point.

[Short description explaining the high-level reason for the pull request]

## Checklist

- [ ] PR has an informative and human-readable title (this will be pulled into the release notes)
- [ ] Changes are limited to a single goal (no scope creep)
- [ ] Code passed the pre-commit check & code is left cleaner/nicer than when first encountered.
- [ ] Any _change_ in functionality is tested
- [ ] New functions are documented (with a description, list of inputs, and expected output)
- [ ] Placeholder code is flagged / future TODOs are captured in comments
- [ ] Project documentation has been updated if adding/changing functionality.

### CODE REVIEW

**Review by skrawcz — COMMENTED** [review]
why are you deleting the previous example?

**Comment by elijahbenizzy** [comment]
> why are you deleting the previous example?

I should be -- meant to move it into the subdirectory.

File: hamilton/version.py [hunk 1]
```diff
@@ -1 +1 @@
-VERSION = (1, 27, 3)
```
Comment (skrawcz) [inline hamilton/version.py:1]: this isn't needed?

Comment (skrawcz) [reply hamilton/version.py:1]: should go away with a rebase?

**Review by skrawcz — APPROVED** [review]
notes -- these don't need to be changed?
 - version
 - star counting notebook

**Review by skrawcz — APPROVED** [review]

**Pull request merged**